### PR TITLE
Fix typo in OTTL readme

### PR DESCRIPTION
**Pull request by felixbarny**
Just a quick drive-by typo fix

### CODE REVIEW

**Review by songy23 — APPROVED** [review]

**Comment by felixbarny** [comment]
The tests have succeeded now but I can't merge the PR. @songy23, could you please hit the merge button?

**Comment by songy23** [comment]
👋 A maintainer will merge your PR (I don't have permission unfortunately)

**Review by crobert-1 — APPROVED** [review]

**Pull request merged**
